config/config.go

```diff
@@ -560,11 +563,16 @@ func validate(c *Config) error {
 }
 
 func (c *Config) Parse() error {
-	flag.Parse()
+	return c.ParseArgs(os.Args[0], os.Args[1:])
+}
+
+func (c *Config) ParseArgs(progname string, args []string) error {
+	c.flags.Init(progname, flag.ExitOnError)
+	c.flags.Parse(args)
```

Comment:
```go
err := c.flags.Parse(args)
if err != nil {
  return err
}
```
or at least hint that we ignore it 
```go
_ = c.flags.Parse(args)
```

Reply:
Okay, I missed this comment somehow: https://github.com/zalando/skipper/pull/2248#discussion_r1137497631
Let's returned error for the possible future use case when `ContinueOnError` would be used.